admin/app/jobs/SurgingSportEmailJob.scala

```diff
@@ -0,0 +1,80 @@
+package jobs
+
+import common.{Edition, ExecutionContexts, Logging}
+import conf.Configuration.commercial._
+import contentapi.ContentApiClient
+import contentapi.ContentApiClient._
+import model.{Content, ContentType}
+import ophan.SurgingContentAgent
+import services.EmailService
+
+import scala.concurrent.Future
+
+case class SportsSurging(
+                          id: String,
+                          pageViewCount: Int,
+                          content: Option[ContentType]) {
+
+  val sportsTargetedKeywords = List("sport/sport","sport/rio-2016","sport/olympics-general")
+
+  private val matchesKeyword = content.exists(_.tags.keywordIds.exists(keyword =>
+    sportsTargetedKeywords.contains(keyword)
+  ))
+
+  val isSportsSurgingContent: Boolean = matchesKeyword && pageViewCount > 500
+}
+
+case class SurgingSportEmailJob(emailService: EmailService) extends Logging with ExecutionContexts {
+
+  private val subject = "New surging sports content"
+
+  def run() : Future[Unit] = {
+    val futureEmail: Option[Future[Unit]] = for {
+      adTech <- adTechTeam
+      surgingContentEmail <- surgingContentTeam
+    } yield {
+
+      getAllSportsSurgingContent
+        .map( surgingContent => {
```

Comment:
I think you can use `onSuccess` here since you want to ultimately return `Unit`, which means you can get rid of the final `.map(_ => ())`